3rdparty/preact/compat/memo.ts

```diff
@@ -8,7 +8,7 @@ import { shallowDiffers } from './util';
  * @param {(prev: object, next: object) => boolean} [comparer] Custom equality function
  * @returns {import('./internal').FunctionComponent}
  */
-export function memo(c, comparer) {
+export function memo(c, comparer?: any) {
```

Comment:
I was unsure of how the `.d.ts` files are being generated, so I added the type annotation to both the source and the definition file.

Reply:
Ah yup, looks good. It's generated by tsc with `"declaration": true`.